docs/deploy-cephfs.md

```diff
@@ -163,7 +163,7 @@ for more information.
 kubectl create -f csi-cephfsplugin-provisioner.yaml
 ```
 
-Deploys stateful set of provision which includes external-provisioner
+Deploys daemonset of provision which includes external-provisioner
```

Comment:
Statefulset is replaced with deployment not daemonset

Reply:
Oops, sorry. It's a mistype. Fixed.